notebooks/features/vw/Vowpal Wabbit - Classification using VW-native Format.ipynb

```diff
@@ -0,0 +1,205 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Comment:
Maybe combine the two classification notebooks, and build two mode3ls, one for native and one for SparkML vtors, and provide a little text describing when to use one or the other

Reply:
(I know you already do this in the overview but readers may not see it there)

Reply:
done